config/config.go

```diff
@@ -308,14 +308,14 @@ type FacadeConfig struct {
 
 // StateTriesConfig will hold information about state tries
 type StateTriesConfig struct {
-	SnapshotsEnabled                   bool
-	AccountsStatePruningEnabled        bool
-	PeerStatePruningEnabled            bool
-	CollectStateChangesEnabled         bool
-	CollectStateChangesWithReadEnabled bool
-	MaxStateTrieLevelInMemory          uint
-	MaxPeerTrieLevelInMemory           uint
-	StateStatisticsEnabled             bool
+	SnapshotsEnabled            bool
+	AccountsStatePruningEnabled bool
+	PeerStatePruningEnabled     bool
+	StateChangesDataAnalysis    bool
```

Comment:
this change needs to be reflected also in config.toml

Reply:
done